react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -141,6 +142,7 @@ class OverflowMenu extends PureComponent<Props, State> {
                 <AudioOnlyButton { ...buttonProps } />
                 {!_reactionsEnabled && !toolbarButtons.has('raisehand') && <RaiseHandButton { ...buttonProps } />}
                 <Divider style = { styles.divider } />
+                <ToggleSelfViewButton { ...buttonProps } />
```

Comment:
Why is this here too? We don't have it there on web do we? This feature is obscure enough as it is, so I'd keep it in the thumbnail menu only.